Tools/scripts/summarize_stats.py

```diff
@@ -211,12 +212,12 @@ def gather_stats(input):
     else:
         raise ValueError(f"{input:r} is not a file or directory path")
 
-def extract_opcode_stats(stats):
+def extract_opcode_stats(stats, prefix):
     opcode_stats = collections.defaultdict(dict)
     for key, value in stats.items():
-        if not key.startswith("opcode"):
+        if not key.startswith(prefix):
             continue
-        name, _, rest = key[7:].partition("]")
+        name, _, rest = key[len(prefix) + 2:].partition("]")
```

Comment:
Is this correct? The old value was `7`, but `len("opcode") + 2` is `8`.

Reply:
Good catch.  It seems to not matter, but might as well revert it just in case.